modules/ROOT/nav.adoc

```diff
@@ -29,10 +29,18 @@ include::{partialsdir}/attributes.adoc[]
 // METRICS
 ** xref:metrics/index.adoc[{metrics-service}]
 *** xref:metrics/provisioning.adoc[Provisioning]
-*** xref:metrics/binding.adoc[Binding]
-*** xref:metrics/configuring-dev-env-metrics.adoc[Configuring a Dev environment]
-*** xref:metrics/analysing-app-usage.adoc[Analysing App Usage]
-*** xref:monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+*** Client metrics
+**** xref:metrics/binding.adoc[Binding]
+**** xref:metrics/configuring-dev-env-metrics.adoc[Configuring a Dev environment]
+**** xref:metrics/analysing-app-usage.adoc[Analysing App Usage]
+*** Service metrics
+**** xref:metrics/monitoring-keycloak.adoc[Monitoring {keycloak-service}]
+*** More...
+**** xref:metrics/giving-an-openshift-user-access-to-metrics.adoc[Sharing access]
+**** xref:metrics/making-mobile-services-discoverable-by-metrics.adoc[Auto-discovery]
+**** xref:metrics/manually-adding-services-to-metrics.adoc[Manual discovery]
+**** xref:metrics/using-dashboards.adoc[Using dashboards]
+**** xref:metrics/using-mobile-services-dashboard.adoc[Mobile Services dashboard]
```

Comment:
I think we should remove the `To be done in the future` section from this file.

Reply:
I will actually remove that file completely since it explains the dashboard for Keycloak. ...which is already explained in Monioring Identity Management document.